packages/local-cli/link/link.js

```diff
@@ -99,16 +99,23 @@ const linkAssets = (platforms, project, dependency) => {
   log.info('Assets have been successfully linked to your project');
 };
 
+type FlagsType = {
+  platforms: string,
+};
+
 /**
  * Links specified package.
  *
  * @param args [packageName]
  */
-function link([rawPackageName]: [string], ctx: ContextT) {
+function link([rawPackageName]: [string], ctx: ContextT, opts: FlagsType) {
   let platforms;
   let project;
   try {
     platforms = getPlatforms(ctx.root);
+    if (opts.platforms) {
+      platforms = pick(platforms, opts.platforms);
```

Comment:
would be better to move this logic into `getPlatforms` so we don't have to overwrite

Reply:
So that was my first idea, pass a second argument to `getPlatforms` to somehow blacklist some platforms. On second though, maybe another file will need these `getPlatforms` and it might be better it stays with same functionality to always return all? Then the consumer (like `link`) can decide what to do with it. You know, to know overcomplicate the function.

Reply:
I think it would be perfectly valid for `getPlatforms` to decide which platforms to return based on "options" argument :)